FusionIIIT/applications/counselling_cell/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
 
+from applications.academic_information.models import Student
 # Create your models here.
+
+
+POSTIONS= (
+    ('pos1', 'Student'),
+    ('pos2', 'Student_Guide'),
+    ('pos3', 'Student_Coordinator'),
+)
+STATUS= (
+    ('status1', 'Unresolved'),
+    ('status2', 'Resolved'),
+    ('status3', 'InProgress'),
+)
+
+ISSUE_CATEGORY= (
+    ('category1', 'Academics'),
+    ('category2', 'Personal'),
+    ('category3', 'Others'),
+)
+
+FAQ_CATEGORY= (
+    ('category1', 'Academics'),
+    ('category2', 'Personal'),
+    ('category3', 'Others'),
+)
```

Comment:
Create separate tables for issue category and FAQ category as these may change over time, so it is not a good idea to hardcode these.